providers/hetznercloud/flags.go

```diff
@@ -16,16 +16,9 @@ var ProviderFlags = []cli.Flag{
 		FilePath: os.Getenv("WOODPECKER_HETZNERCLOUD_API_TOKEN_FILE"),
 		Category: category,
 	},
-	&cli.StringFlag{
-		Name:     "hetznercloud-location",
-		Value:    "nbg1",
```

Comment:
Isn't this breaking as it would throw a unknown argument error for location now?

Reply:
Well ok for me but then Ill tag an 1.0 release after merge because 0.x is useless anyway.

Reply:
We could also do some announcements on SoMe to make people aware of the autoscaler and all the supported providers.

Reply:
Honestly this is too overengineered for me. Version updates should be checked. For 0.x versions even more. Thats the case for every single software. Feel free to still do some social media posts.

Reply:
I have added the flag back, use it as fallback and added a deprecation notice.